test/e2e/dynamic-import-app-group-preloading/fixture/src/split-shared-ab.js

```diff
@@ -0,0 +1,977 @@
+// @flow
+
+import React from 'react';
+
+export default () => (
+  <div>
+    <span>split-shared-ab</span>
+    {reallyBigTextSoThisFileHasALargeFilesize}
+  </div>
+);
+
+var reallyBigTextSoThisFileHasALargeFilesize = `
```

Comment:
Just leaving a comment here so when webpack changes down the road future generations have a place to laugh at us. Sorry! 😅